src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanExpressionCheck.java

```diff
@@ -39,6 +39,31 @@
  * <pre>
  * &lt;module name=&quot;SimplifyBooleanExpression&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *
+ *  public void bar(){
+ *
+ *   boolean a, b;
+ *   Foo c, d, e;
+ *
+ *   if(!false){}; // violation, can be simplified to true
+ *
+ *   if(a == true){}; // violation, can be simplified to a
+ *   if(a == b){}; // OK
+ *   if(a == false){}; // violation, can be simplified to !a
+ *   if(!(a != true)){}; // violation, can be simplified to a
```

Comment:
Please put white spaces before and after parentheses here and above

Reply:
Ok.

Reply:
@wilcoln , please avoid extra comments in review items, please reply to it with "done" or any disagreement or clarification request.

Reply:
@wilcoln , please add space after each IF.

Reply:
@wilcoln please correct item above.